### ISSUE TITLE
[ Docs ] Update Readme to Include Status of current work

### ISSUE THREAD

**Issue by TreavVasu** [ECellNitrr/ECellOfficialApp#5]
### Is your feature request related to a problem? Please describe.
In order to track the progress of the project more efficiently, we need a better way to view progress. 

### Describe the solution you'd like
1. Follow Contribute.md
2. Update the Readme.md 
3. Mention filename as well as File status present like: 
   ``` 
      screen:
                 Home [X]
                 AboutUs [X]
                .
                .
                .
      Cubit:
                Speaker [X]
                Team [X]
                ... 
      API file:
                Speaker [X]
                Hon'ble Mention [  ]
                .
                .
   ```

### Additional context
- Reach out in case of confusion
- No need to stress if it looks complex just to see which files are cubit which are screen/ui and which are API files and mark accordingly 
- No need to Update anything else in Readme.MD
- [ X] - means a ✅, O, or a flag that states it's done
- [   ] - means incomplete or work left to do 

### Priority
- High
### Complexity
- Easy

**Comment by TreavVasu** [comment]
Divide the work among yourself or either one of you skip(Unassign) this issue and work on the other one [Update]

**Comment by TreavVasu** [comment]
Closed by #8
